benchmark/bench_CQRRP/CQRRP_pivot_quality.cc

```diff
@@ -82,16 +82,16 @@ static void R_norm_ratio(
     std::iota(all_data.J.begin(), all_data.J.end(), 1);
     //RandLAPACK::hqrrp(m, n, all_data.A.data(), m, all_data.J.data(), all_data.tau.data(), b_sz,  (d_factor - 1) * b_sz, 0, 0, state, (T*) nullptr);
     lapack::geqp3(m, n, all_data.A.data(), m, all_data.J.data(), all_data.tau.data());
-    std::vector<T> R_norms_HQRRP = get_norms<T, RNG>(all_data);
+    std::vector<T> R_norms_HQRRP = get_norms<T>(all_data);
     printf("\nDone with HQRRP\n");
 
     // Clear and re-generate data
-    data_regen<T, RNG>(m_info, all_data, state);
+    data_regen<T>(m_info, all_data, state);
 
     printf("\nStarting CQRRP\n");
     // Running CQRRP
     CQRRP_blocked.call(m, n, all_data.A.data(), m, d_factor, all_data.tau.data(), all_data.J.data(), state);
-    std::vector<T> R_norms_CQRRP = get_norms<T, RNG>(all_data);
+    std::vector<T> R_norms_CQRRP = get_norms<T>(all_data);
```

Comment:
``T`` can be inferred from ``all_data``.

Reply:
Fixed.